tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>Shouldn't we omit this way of creating a <code>DetuningMap</code>? I would say it's preferable to only show how it is created with <code>define_detuning_map()</code> instead </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Okay I agree I think not presenting it will avoid many mistakes/questions from users.</p>